README.md

```diff
@@ -47,6 +47,10 @@ $ apt-get install zfsutils-linux
 
 Go to each node and create the ZFS Pool, which will be used for provisioning the volumes. You can create the Pool of your choice, it can be striped, mirrored or raidz pool.
 
+Configure the custom topology keys (if needed). This can be used for many purposes like if we want to create the PV on nodes in a particuler zone or building. We can label the nodes accordingly and use that key in the storageclass for taking the scheduling decesion:
```

Comment:
Is this fixed in 0.7 or 0.6? Should i wait to merge this?

Reply:
0.7, we can merge it.